cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -67,6 +67,15 @@ const (
 	DefaultProxyBindAddressv6 = "::"
 	// KubeproxyKubeConfigFileName defines the file name for the kube-proxy's KubeConfig file
 	KubeproxyKubeConfigFileName = "/var/lib/kube-proxy/kubeconfig.conf"
+
+	// DefaultAuditPolicyLogDir is the default directory where the audit system will write
+	DefaultAuditPolicyLogDir = "/var/log/kubernetes"
```

Comment:
How about creating a sub-dir? This is OK anyway.

Reply:
@xiangpengzhao - Do you mean /var/log/kubernetes/audit ?

Reply:
@timothysc exactly :)